pkg/vdr/key/creator.go

```diff
@@ -50,15 +46,7 @@ func (v *VDR) Create(keyManager kms.KeyManager, didDoc *did.Doc,
 	)
 
 	if len(didDoc.VerificationMethod) == 0 {
-		keyType, err = extractKeyTypeFromOpts(createDIDOpts)
-		if err != nil {
-			return nil, err
-		}
-
-		err = createNewKeyAndVerificationMethod(didDoc, keyType, keyManager)
-		if err != nil {
-			return nil, err
-		}
+		return nil, fmt.Errorf("verification method is empty")
```

Comment:
Note `verificationMethod` is optional in generic DID documents: https://w3c.github.io/did-core/#verification-methods

(DID documents may inline all keys in the other sections).